R/dal.R

```diff
@@ -1305,8 +1306,16 @@ load_clean_dist_sp <- function(azcontainer = suppressMessages(get_azure_storage_
                                end.year = lubridate::year(Sys.Date()),
                                st.year = 2000,
                                data.only = F,
-                               type = NULL) {
-  cli::cli_alert_info("Loading district spatial files")
+                               type = NULL,
+                               version = "standard") {
+
+  if(version == "dev"){
+    fp <- "GID/PEB/SIR/Data/spatial_dev/global.dist.rds"
```

Comment:
This is completely fine and obviously works. May be worthwhile to think of a less "hard-coded" method to do this for future work. Should be fine for this one!